tests/backend/unit/test_logging.py

```
@@ -0,0 +1,1315 @@
+        Returns:
+            None
+        
+        Example:
+            This test defines a FastAPI app with a slow endpoint, applies the logging middleware, and verifies that the logged request duration is at least 100 milliseconds for a simulated slow request.
+        
+        Generated by CodeRabbit
+        """
+        app = FastAPI()
+        
+        @app.get("/slow")
+        async def slow_endpoint():
```

Ruff (0.11.9)

Missing return type annotation for private function `slow_endpoint`

(ANN202)